src/Commands/FormatCommandCommon.cs

```diff
@@ -345,9 +345,9 @@ internal static bool TryLoadMSBuild([NotNullWhen(returnValue: true)] out string?
         {
             try
             {
-                // Get the latest .NET 6 SDK instance.
+                // Get the latest .NET SDK instance.
```

Comment:
do we take this code path when running as a global tool? it seems like we would want "whatever SDK is launching us" in that case, right?

Reply:
We always take this path. The CLI and included tooling is typically updated to be the highest versioned SDK.